### ISSUE TITLE
git usage in package.json behind proxy

### ISSUE THREAD

**Issue by andriuwe4ka** [fin-hypergrid/fin-hypergrid-filter-and-sort-demo#2]
While it is basically ok to use direct links in package.json ([one](https://github.com/fin-hypergrid/fin-hypergrid-filter-and-sort-demo/blob/master/package.json#L21), [two](https://github.com/fin-hypergrid/fin-hypergrid-filter-and-sort-demo/blob/master/package.json#L22)), it is useless for those who are behind proxy.
Not a big deal, but could be easier for end users.

**Comment by Dwaynekj** [comment]
Ah good point. We wanted to split out the demos. So what you could do is fork, fetch the package jsons and resolve locally on your file system?

**Comment by andriuwe4ka** [comment]
yes, i did almost as you described, but for me it is demo and must work from box.
anyway, it is not basically issue at all, it is more about making demo user/developer friendly ;-)

**Comment by Dwaynekj** [comment]
Any ideas on how we could refactor it without making one repo?

**Comment by Dwaynekj** [comment]
how about this repo
https://github.com/fin-hypergrid/old-addons/blob/master/fin-hypergrid-filtering-demo/package.json

Will it suffice?
